File: java-advent-of-code/src/test/java/advent/twenty_eighteen/Day04ReposeRecordTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day04ReposeRecordTest {
    @Test
    public void checkReal() {
        Day04ReposeRecord im = new Day04ReposeRecord("/twenty_eighteen/Day04-ReposeRecord-input.txt");

        assertEquals(19830, im.getPart1Answer().intValue());
        assertEquals(43695, im.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        Day04ReposeRecord im = new Day04ReposeRecord("/twenty_eighteen/Day04-ReposeRecord-example#1.txt");

        assertEquals(240, im.getPart1Answer().intValue());
        assertEquals(4455, im.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day04ReposeRecordTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day04ReposeRecordTest  extends FlatSpec with Matchers {
  "2018 Day 04 - Input" should "calculate answers" in {
    val dp = new Day04ReposeRecord("/twenty_eighteen/Day04-ReposeRecord-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day04ReposeRecord("/twenty_eighteen/Day04-ReposeRecord-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
